cdc/scheduler/internal/v3/keyspan/reconciler.go

```diff
@@ -69,6 +71,7 @@ func NewReconciler(
 		changefeedID: changefeedID,
 		config:       config,
 		splitter: []splitter{
+			// write splitter has the highest priority.
```

Comment:
why it is the highest one ?

Reply:
This is by design. When the user enables the feature to split tables based on input volume, it will have higher priority.